src/applications/facility-locator/tests/e2e/search.cypress.spec.js

```diff
@@ -30,7 +30,6 @@ Cypress.Commands.add('verifyOptions', () => {
 
 describe('Facility search', () => {
   before(function() {
-    if (!Cypress.env('CIRCLECI')) this.skip();
```

Comment:
I think we need to keep this as-is. This particular spec causes browser crashes on Jenkins but not on CircleCI, which is why I disabled it on Jenkins.

Reply:
Are you sure? All of the tests passed in Jenkins in the last [run](http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fvets-website/detail/fl_manual_geolocation/16/pipeline/167).

Reply:
These tests are flaky, so they do not fail on all runs. The browser crashes happen in roughly one out of every 15-20 runs.

Reply:
This is a critical test for my team.

I've heard that Ops is moving away from CircleCI.

We've upgraded Cypress recently, so maybe the stability will improve?

Can we try re-enabling it, and if it continues to be a problem, revisit the issue?

Reply:
The Cypress upgraded improved how the failure is handled -- before, the test would hang indefinitely whereas now Cypress catches the browser crash and moves on to the next test. I tried multiple methods to get these tests to stabilize, but none were effective. I think we should enable the tests in a separate PR that explores any other methods to fix the failures. At this point the only fix I'm aware of is to allot more resources to our Jenkins machines. @pjhill wanted to get your thoughts here

Reply:
Good discussion -- I'm not _too_ hopeful that the recent minor version updates to Cypress would have resolved the browser crash issue, but It's probably prudent to test that from time to time. We also need to consider the importance of E2E tests to the QA processes of VFS teams.

With that said, I agree with Darius -- @mpelzsherman would you be comfortable owning the PR to re-enable the test and troubleshoot flakiness?

Reply:
I will add a ticket to our backlog and let @mmiddaugh prioritize.

Flaky tests will be less of an issue once we have isolated the applications. I think this is on the VSP roadmap for Q2.

Reply:
@mpelzsherman The one thing I was not able to test was mocking the Mapbox call like you have in this PR since I'm not familiar with the API. My suggestion would be to try mocking out the Mapbox calls to see if it fixes the failures. Map rendering and map animations are causing the crashes.

Reply:
[Story](https://github.com/department-of-veterans-affairs/va.gov-team/issues/20178) created